src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -9726,7 +9736,14 @@ bool StorageReplicatedMergeTree::createEmptyPartInsteadOfLost(zkutil::ZooKeeperP
                 }
             }
 
-            getCommitPartOps(ops, new_data_part);
+            /// Two replicas may try to commit an empty part simultaneusly,
+            /// so some lost part may be counter twice in lost_part_count. Avoid that by using the part name as deduplication id.
```

Comment:
> Avoid that by using the part name as deduplication id

The comment is no longer precise. `getOpsToCheckPartChecksumsAndCommit` is doing all that work now, right?